File: src/Bloemert.Data.Entity.Skills/Entity/Sector.cs
using Bloemert.Data.Core;
using System;

namespace Bloemert.Data.Entity.Skills.Entity
{
	public class Sector : BaseEntity
	{
		public virtual int Seqno { get; set; }

		public virtual string Name { get; set; }

		public virtual string Description { get; set; }

	}
}

File: src/Einstein.WebUI/ClientApp/components/skills/sectors/models/Sector.js
﻿
export class Sector {
	constructor({
		seqno = 0,

		status = {
			valid: true,
			added: false,
			modified: false,
			removed: false,
			loaded: false
		},

		id = -1,
		deleted = false,

		name = '',
		description = '',

		} = {}) {

		this.id = id;
		this.deleted = deleted;

		this.seqno = seqno;
		this.status = status;

		this.name = name;
		this.description = description;
	}
}

export function createSector(data) {
	return Object.freeze(new Sector(data));
}